src/commands/karma.ts

```diff
@@ -0,0 +1,27 @@
+import { Command } from 'discord-akairo';
+import { Message } from 'discord.js';
+import Repository from '../repository/Repository';
+import { UserKarmaInfo } from '../types';
+import { getEmbedFromUserKarma } from '../util';
+
+export default class LeaderboardCommand extends Command {
+    constructor() {
+        super('karma', {
+           aliases: ['karma']
+        });
+    }
+
+    async exec(message: Message) {
+        const { author } = message;
+        const userKarma: UserKarmaInfo[] | undefined = await Repository.getUserKarmaInfo(author.id);
```

Comment:
TypeScript has type inference, LHS need not be typed as RHS can help compiler determine type of `userKarma`

Reply:
Yeah type will be determined, but what if someone changes return type of `getUserKarmaInfo` then ts will show error here on the same line itself, rather than where `userKarma` is used. And it looks more readable imo.